dgraphtest/cluster.go

```diff
@@ -76,6 +79,27 @@ type GraphQLResponse struct {
 	Extensions map[string]interface{} `json:"extensions,omitempty"`
 }
 
+type Location struct {
+	Line   int `json:"line,omitempty"`
+	Column int `json:"column,omitempty"`
+}
+
+type gqlError struct {
+	Message    string                 `json:"message"`
+	Locations  []Location             `json:"locations,omitempty"`
+	Path       []interface{}          `json:"path,omitempty"`
+	Extensions map[string]interface{} `json:"extensions,omitempty"`
+}
+
+type gqlErrorList []*gqlError
+
+type LicenseResponse struct {
```

Comment:
can we not use dgraphtest.GraphqlResponse instead here?

Reply:
I checked, the structures are different, so, did not change.